code/datums/outfit.dm

```diff
@@ -197,15 +197,15 @@
 	var/stored_data = get_json_data()
 	var/json = json_encode(stored_data)
 	//Kinda annoying but as far as i can tell you need to make actual file.
-	var/f = file("data/TempOutfitUpload") 
+	var/f = file("data/TempOutfitUpload")
 	fdel(f)
 	WRITE_FILE(f,json)
 	admin << ftp(f,"[name].json")
 
 /datum/outfit/proc/load_from(list/outfit_data)
 	//This could probably use more strict validation
 	name = outfit_data["name"]
-	uniform = text2path(outfit_data["uniform"])
+	uniform = outfit_data[uniform]	//already a path
```

Comment:
What's the reasoning here? I think this has to do with loading custom outfits from .json in which case it will not already be a path

Reply:
You're right, when I was looking through this all initially I was under the impression all of the outfit data was stored as text then converted into their objects through this and the text2path function, but I was mistaken.